docs/platform/SEPES component overview.md

```diff
@@ -92,8 +97,7 @@ Protocol: Uses http, so have to be driven in the docker. Link documentation at O
 
 ## Azure AD ##
 
-Used for autentication. Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
-
+Used for autentication. Link for Azure AD use. https://docs.microsoft.com/nb-no/azure/active-directory/
```

Comment:
Used for autentication. Link for Azure AD use. https://docs.microsoft.com/azure/active-directory/